### PR TITLE
[server] Disable new setup modal as long as it's an dead end

### PR DESCRIPTION
**Pull request by geropl**
## Description

Enables testing of WEB-27

## Related Issue(s)

Fixes WEB-27

#### Preview status

gitpod:summary

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold

### CODE REVIEW

**Comment by geropl** [comment]
@AlexTugarev Could you have a quick look? This is meant to unblock testing of Dedicated for the next 2-3 days, until we have the first version of the setup flow.

File: components/server/src/workspace/gitpod-server-impl.ts [hunk 1]
```diff
@@ -621,9 +624,10 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         }
 
         // A session is required to continue with an activated onboarding flow.
-        if (this.showOnboardingFlowCondition?.value) {
-            throw new ResponseError(ErrorCodes.ONBOARDING_IN_PROGRESS, "Dedicated Onboarding Flow in progress.");
-        }
+        // TODO(gpl): Re-enable once we have an onboarding setup view
+        // if (this.showOnboardingFlowCondition?.value) {
+        //     throw new ResponseError(ErrorCodes.ONBOARDING_IN_PROGRESS, "Dedicated Onboarding Flow in progress.");
```
Comment (AlexTugarev) [inline components/server/src/workspace/gitpod-server-impl.ts:629]: @geropl did you try to just comment out the line, leaving the rest in place? The effect would be to not trigger `ONBOARDING_IN_PROGRESS`, which is what I read from the issue. 

That would also be a good change to validate that feature flags do work in the cell, no?

Comment (AlexTugarev) [reply components/server/src/workspace/gitpod-server-impl.ts:629]: Why? If `enableDedicatedOnboardingFlow` FF is set _and_ this line is disabled, the preceding `SETUP_REQUIRED` is not thrown, see. `&& !this.showOnboardingFlowCondition?.value`

Comment (AlexTugarev) [reply components/server/src/workspace/gitpod-server-impl.ts:629]: That only will work if FFs are working in the cell, which as I understood, is already the case.

Comment (geropl) [reply components/server/src/workspace/gitpod-server-impl.ts:629]: Moving forward to get unblocked here :+1:

**Pull request merged**